apps/agora/api/src/components/genes.ts

```diff
@@ -164,7 +164,7 @@ export async function genesRoute(req: Request, res: Response, next: NextFunction
   try {
     const result = await getGenes(<string | string[]>req.query.ids);
     setHeaders(res);
-    res.json({ items: result });
+    res.json(result);
```

Comment:
in the future, it may be good to create consistency as returning an object with an items property could be replaced with just returning the array